PoseEstimation-TFLiteSwift/VideoCapture.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  VideoCapture.swift
+//  Awesome ML
+//
+//  Created by Eugene Bokhan on 3/13/18.
+//  Copyright © 2018 Eugene Bokhan. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+import CoreVideo
+
+public protocol VideoCaptureDelegate: class {
+    func videoCapture(_ capture: VideoCapture, didCaptureVideoFrame: CVPixelBuffer?, timestamp: CMTime)
+}
+
+public class VideoCapture: NSObject {
+    public var previewLayer: AVCaptureVideoPreviewLayer?
+    public weak var delegate: VideoCaptureDelegate?
+    public var fps = 30
+    
+    let captureSession = AVCaptureSession()
+    let videoOutput = AVCaptureVideoDataOutput()
+    let queue = DispatchQueue(label: "com.tucan9389.camera-queue")
+    
+    var lastTimestamp = CMTime()
+    
+    public func setUp(sessionPreset: AVCaptureSession.Preset = .vga640x480,
+                      cameraPosition: AVCaptureDevice.Position = .back,
+                      completion: @escaping (Bool) -> Void) {
+        self.setUpCamera(sessionPreset: sessionPreset,
+                         cameraPosition: cameraPosition,
+                         completion: { success in
+            completion(success)
+        })
+    }
+    
+    func setUpCamera(sessionPreset: AVCaptureSession.Preset, cameraPosition: AVCaptureDevice.Position, completion: @escaping (_ success: Bool) -> Void) {
+        
+        captureSession.beginConfiguration()
```

Comment:
In Apple docs,

They recommend using other dispatchQueue to developer for AVCaptureSession Configure.
Because 'AVCaptureSession Configure' is blocking operation.

I share the links.

https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/avcam_building_a_camera_app
: in this page, please search a keyword 'block'.

https://developer.apple.com/documentation/avfoundation/avcapturesession
: please read 'important' part.

Reply:
Thanks for the nice reference and guide. It's really helpful for me. 
Actually blocking the camera session queue was my intention because I want to block capturing during inference time for performance issues. I think it is a little trick. But in this app, it seems no issue. Do you have any predictable side effect for the reason?

Reply:
@syjdev 
I'll change it in the next PR. Thank you.